src/components/EventCardsGridContainer/index.js

```diff
@@ -77,23 +88,99 @@ class EventCardsGrid extends React.Component {
     } = nextProps;
 
     if (eventStatusIndex !== this.props.eventStatusIndex
-      || sortBy !== this.props.sortBy
-      || syncBlockNum !== this.props.syncBlockNum) {
-      this.executeGraphRequest(eventStatusIndex, sortBy);
+      || sortBy !== this.props.sortBy) {
+      this.executeGraphRequest(eventStatusIndex, sortBy, LIMIT, SKIP);
+      this.setState({ skip: 0 });
+    }
+    if (syncBlockNum !== this.props.syncBlockNum && this.props.syncBlockNum !== 0) {
+      this.setState({ hint: true });
+      setTimeout(() => {
+        this.setState({ hint: false });
+      }, 3000);
```

Comment:
what is this delayed action for? is it supposed to be a loading hint?

Reply:
it's for the load new hint, say a new block come, maybe new data comes, refresh to get the new data, but the user can ignore it.

Reply:
The user cannot manually refresh on the electron version. This was causing an error with routing and would should an error page if they are in the detail page.

The page should reload automatically on a new block as we should know what the current page is by the limit and skip.